FirstClasses/multiples/src/multiples.erl

```diff
@@ -1,5 +1,39 @@
 -module(multiples).
 
 -export([thousand_sum/0]).
+-export([generate_input_collection/1]).
+-export([filter_collection/1]).
 
-thousand_sum() -> 1.
+thousand_sum() -> 
+    FilterCollection = filter_collection(generate_input_collection(1000)),
+    lists:sum(FilterCollection).
+
+generate_input_collection(Lenght) ->
+    case Lenght of
+        0  ->
+           [];
+        L when L - 1 == 0 ->
+            1;
+        _ ->
+            lists:seq(1, Lenght - 1)
+    end.
+
+filter_collection(List) ->
+    case List of
+        [] ->
```

Comment:
You do not need a case here at all, when you apply ANY filter to an empty list (`[]`) it will always return an empty list.
In other words, the following expresion is true for any `Pred` `lists:filter(Pred, []) -> [].`
;)
Btw if you look into the implementation is requires `Pred` to be a one argument function.
https://github.com/erlang/otp/blob/6ccbbb0b026b46aaa006922376e76fa143910b34/lib/stdlib/src/lists.erl#L1285